frontend/src/components/admin/menu/DictionaryManagement.js

```diff
@@ -200,56 +392,95 @@ function DictionaryManagement() {
+                      marginBottom: "1rem",
                     }}
-                  >
-                    <SelectItem text="" />
-                    {categoryDescription.map((description) => (
-                      <SelectItem
-                        key={description.id}
-                        value={description.id}
-                        text={description.description}
-                      />
-                    ))}
-                  </Select>
+                  />
                   <TextInput
                     id="dictEntry"
                     labelText="Dictionary Entry"
+                    value={dictionaryEntry}
                     onChange={(e) => setDictionaryEntry(e.target.value)}
                     style={{
                       marginBottom: "1rem",
                     }}
                   />
-                  <TextInput
-                    data-modal-primary-focus
+                  <Dropdown
                     id="isActive"
-                    labelText="Is Active"
-                    onChange={(e) => setIsActive(e.target.value)}
+                    type="default"
+                    items={yesOrNo}
+                    titleText="Is Active"
+                    itemToString={(item) => (item ? item.id : "")}
+                    onChange={({ selectedItem }) => {
+                      setIsActive(selectedItem);
+                    }}
+                    selectedItem={isActive}
+                    size="md"
                     style={{
                       marginBottom: "1rem",
                     }}
                   />
                   <TextInput
                     id="localAbbrev"
                     labelText="Local Abbreviation"
+                    value={localAbbreviation}
                     onChange={(e) => setLocalAbbreviation(e.target.value)}
                     style={{
                       marginBottom: "1rem",
                     }}
                   />
                 </Modal>
-                <Button disabled={true} type="submit">
-                  <FormattedMessage id="admin.page.configuration.formEntryConfigMenu.button.modify" />
-                </Button>{" "}
-                <Button kind="tertiary" disabled={true} type="submit">
+                <Button
+                  disabled={modifyButton}
+                  onClick={handleDeactivation}
+                  type="submit"
+                >
                   <FormattedMessage id="admin.page.configuration.formEntryConfigMenu.button.deactivate" />
                 </Button>
               </Column>
+              <Column
+                lg={16}
+                md={8}
+                sm={4}
+                style={{
+                  display: "flex",
+                  flexDirection: "column",
+                  alignItems: "center",
+                  gap: "10px",
+                }}
+              >
+                <Link>
+                  Showing {toRecordCount} - {fromRecordCount} of{" "}
```

Comment:
The button is disabled if paging is 1 and `startingRecNo` is less than or equal to 21.
The button is also disabled if `startingRecNo` is 1 or less.